packages/translator/src/resolvers/openAI.ts

```diff
@@ -119,7 +119,7 @@ export const openAIResolver = ({
           }
 
           for (const text of translatedChunk) {
-            if (typeof text !== 'string') {
+            if (text !== null && typeof text !== 'string') {
```

Comment:
We need to check if it's `undefined` as well. I would change to:

```
            if (text && typeof text !== 'string') {
```

Reply:
good point, thank you